src/types/FilterType.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterType {
```

Comment:
Consider adding a comment above the enum declaration to describe its purpose in the application. 📝